build/common.mk

```diff
@@ -168,9 +167,8 @@ test_fast::
 
 install::
 	$(call STEP_MESSAGE)
-	@mkdir -p $(PULUMI_BIN)
-	@mkdir -p $(PULUMI_NODE_MODULES)
-	@mkdir -p $(PULUMI_NUGET)
+	@mkdir -p $(GOBIN)
```

Comment:
I think we should check that this doesn't exist - we don't want to kill anyones GOBIN

Reply:
I'm pretty sure that when `mkdir` is called on a folder that already exists, nothing happens. That is certainly true on macOS.

Reply:
🆗